packages/zipkin-transport-aws-sqs/test/AwsSqsLogger.test.js

```diff
@@ -0,0 +1,11 @@
+const AwsSqsLogger = require('../src/AwsSqsLogger');
+
+describe('Aws SQS transport', () => {
+  it('should accept to JSON_V2 encoder', () => {
```

Comment:
I think this test name is not anymore accurate isn't?

Reply:
yep, i have to change, as well as the README